MAINTAINERS.md

```diff
@@ -0,0 +1,282 @@
+
+A maintainer *must* also have 2FA enabled on their GitHub account.
+
+#### The Rights of a Maintainer
+
+You may choose to do zero or more of these *at their discretion*:
+
+- Merge pull requests
+- Modify issues (closing, adding labels, assigning them other maintainers, etc.)
+- Modify GitHub [Projects](https://github.com/mochajs/mocha/projects)
+- Cancel builds, restart jobs, or otherwise interact with our CI server(s) 
+- CRUD operations on GitHub integrations
+- Participate in the decision-making process
+- Add new maintainers to the team
+- Tag releases and publish Mocha to npm
+
+> While maintainers have the ability to commit directly to the `master` branch, *this is to be avoided* if any other maintainer could reasonably take issue with the change, or the change affects Mocha's API or output.  For example, a spelling correction in `CHANGELOG.md` may not require a pull request.  A change to a reporter's output most certainly would!  Maintainers are trusted to use their best judgement; if unsure, err on the side of caution.
+
+#### About "Owners"
+
+Some maintainers will have full admin rights to the [mochajs org](https://github.com/mochajs) and/or will have access to publish to npm.
+
+- Those with publish access are expected to use npm's 2FA.
+- This level of access will be granted by the current owners to those maintainers who have earned the project's trust.
+
+## Mocha's Decision-Making Process
+Mocha follows a [consensus-seeking decision-making](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making) process.  In other words, all maintainers attempt to come to agreement.  If that fails, we decide by a simple vote.
+
+Active maintainers will make an effort to solicit feedback from others before making important or potentially controversial decisions.  Given the varying geographical distribution and availability of the maintenance team, we resolve to do the best we can to solicit feedback.
+
+In other words, to have your opinion heard, participate regularly.  The rest of the team won't wait on feedback that isn't necessarily forthcoming!
+
+## Communication
+
+Maintainers will mainly gather in [the mochajs/contributors channel](https://gitter.im/mochajs/contributors).  This is a *public* channel, and *anyone* can join.
+Videoconference (or audio) calls may happen on a regular or irregular basis, as schedules allow. This is mainly because we have Real Lives and time zones suck.
+
+## Labeling Issues & Pull Requests
+
+All new issues will need to be triaged, and pull requests must be examined.  Maintainers must understand [Semantic Versioning](http://semver.org) ("SemVer"), as Mocha follows it strictly.
+
+> If you see an issue or PR that could use some labels, please add them!
+
+The following list is incomplete, but it's better than nothing:
+
+### Semantic Versioning
+
+*All* issues should have one of these three labels:
```

Comment:
Should there be an exception for issues labelled "question" (since those are just support rather than requiring changes)?

Reply:
yes; I'll reword this.  👍